### PR TITLE
Trying to get the mess sorted out. 

### PR DESCRIPTION
**Pull request by ppujol76**
Requested modifications in the encoder part done. Removed modifications done to handle splits in the dataset class. Added Hparam with Imagenet normalization parameters and changed main.py accordingly

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import torch.optim as optim
 from torch.utils.data import DataLoader, random_split
 from torchvision import transforms
+import pandas as pd
```
Comment (amolero6) [inline main.py:7]: Do we need this import here?

**Review by amolero6 — APPROVED** [review]

File: config.json [hunk 2]
```diff
@@ -11,6 +11,8 @@
 		"EMBED_SIZE": 256,
 		"LEARNING_RATE": 1e-3,
 		"IMAGE_SIZE": 224,
-		"LOG_INTERVAL": 2
+		"LOG_INTERVAL": 2,
+		"IMAGE_NET_MEANS": [0.485, 0.456, 0.406],
```
Comment (lucaswerner90) [inline config.json:15]: awesome! thanks for putting this here

File: main.py (hunk 1 above)
Comment (lucaswerner90) [reply main.py:7]: I don't think is needed

File: main.py [hunk 3]
```diff
@@ -26,14 +27,12 @@ def main():
...
-        # Otherwise we could just normalize the values between -1 and 1 using the
-        # standard mean and standard deviation
-        # transforms.Normalize(mean=[0.485, 0.456, 0.406],std=[0.229, 0.224, 0.225]),
-		transforms.Normalize(mean=[0.485, 0.456, 0.406],std=[0.229, 0.224, 0.225]),
+		# The normalize parameters depends on the model we're gonna use
+		# If we apply transfer learning from a model that used ImageNet, then
+		# we should use the ImageNet values to normalize the dataset.
+		# Otherwise we could just normalize the values between -1 and 1 using the 
+		# standard mean and standard deviation
+		transforms.Normalize(mean=hparams['IMAGE_NET_MEANS'],std=hparams['IMAGE_NET_STDS']),
```
Comment (lucaswerner90) [inline main.py:35]: awesome 👍

**Review by lucaswerner90 — APPROVED** [review]

File: config.json (hunk 2 above)
Comment (ppujol76) [reply config.json:15]: Si, lo único que veo que el nombre quería que fuera IMAGENET_MEANS en lugar de IMAGE_NET_MEANS, sin el guión bajo. Pero bueno...

Comment (ppujol76) [reply config.json:15]: Thank you!
